vipps-login-api.md

```diff
@@ -1197,10 +1197,10 @@ to be taken to the client's web page to finalise the flow.
     Authorization: Basic asdkjhasdjhsad=
     Content-Type: application/x-www-form-urlencoded
 
-    scope=name address openid&login_hint=urn:mobilenumber:{mobileNumber}&state=13821s837213bng26e2n61gege26&nonce=21hebdhwqdb7261bd1b23&requested_flow=login_to_webpage&nonce=qwhjewqkheqkwhkqhweqhwekjwh21u3h21he13ew2&code_challenge_method=S256&code_challenge=21je21je2o1j3o21joedj21do1j321e2oi1&redirect_uri=https://merchantwebpage.com/callback
+    requested_flow=login_to_webpage&scope=openid name address&login_hint=urn:mobilenumber:{mobileNumber}&redirect_uri=https://merchantwebpage.com/callback
     ```
 
-    Example response, the response values are typically not used for anything in this flow:
+    Example response, the `auth_req_id` can later be used to connect this login to a token response since the ID token should contain the same `auth_req_id` value.
```

Comment:
```suggestion
    Example response, the `auth_req_id` should be used to connect this login to a token response since the ID token should contain the same `auth_req_id` value.
```